qcengine/programs/gamess/runner.py

```diff
@@ -2,8 +2,10 @@
 
 import copy
 import pprint
+import sys
+import traceback
 from decimal import Decimal
-from typing import Any, Dict, Optional
+from typing import Any, Dict, Optional, Tuple
```

Comment:
These additions don't appear to be used in this module. Is there a reason for adding them?

Reply:
This is the first bite-sized PR from a big block of changes. These imports are used later on, so it seemed easier to avoid missing imports to add them now. I don't mind running an analysis at the end of the PR series to prune away unused imports.